packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -254,6 +240,23 @@ export default function MainFeedLayout({
 
   const hasFiltered = feedName === MainFeedPage.MyFeed && !alerts?.filter;
 
+  /* eslint-disable react/no-children-prop */
+  const feedHeading = {
+    [MainFeedPage.MyFeed]: !hasFiltered ? (
```

Comment:
Optionally maybe we can create a MyFeedHeading component which will do the conditional rendering? (Just to clean this up some more)

Reply:
Good point. Will make this part cleaner. Let me refactor this one.